benchmarks/TFLite/linux-riscv.cmake

```diff
@@ -0,0 +1,73 @@
+# Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+################################################################################
+#                                                                              #
+# Default benchmark configurations                                             #
+#                                                                              #
+# Each suite benchmarks a list of modules with configurations specifying a     #
+# target architecture and runtime characteristics (e.g. threads/cores). These  #
+# benchmarks only configure IREE translation and runtime flags for the target  #
+# architecture and do *not* include any non-default flags. No non-default      #
+# flags should be added here.                                                  #
+#                                                                              #
+################################################################################
+
+set(LINUX_RISCV64_GENERIC_RV64_CPU_TRANSLATION_FLAGS
+  "--iree-input-type=tosa"
+  "--iree-llvm-target-triple=riscv64"
+  "--iree-llvm-target-cpu=generic-rv64"
+  "--iree-llvm-target-abi=lp64d"
+  "--iree-llvm-target-cpu-features=+m,+a,+f,+d,+c"
```

Comment:
See https://github.com/google/iree/blob/52f62b8afdd9663b37f8dd94a247eaa654857bc7/runtime/src/iree/hal/local/elf/testdata/generate.sh#L28-L47 on how to invoke `iree-compile` to generate the dispatchable workload we care about (not the VM control code).

Reply:
Is there a big difference between counting `.vmfb` vs. the embedded dynamic library? The vmfb should include all the embedded dynamic libraries and I assume usually we use `.vmfb` (according to https://google.github.io/iree/building-from-source/riscv/#optional-configuration).

Reply:
If we intend to generate the full bytecode module, we should consider to add `--iree-llvm-debug-symbols=false` to avoid the overhead of including the symbols in the workload library.

If we add `--iree-vm-emit-polyglot-zip` flag then the workload library can be extracted from the bytecode module with `zipinfo`

Reply:
Do we want to track the total size of workload libraries or each of them (there are usually 70~90 libraries in a model)?

The number of workload libraries (dispatches) are frequently changed when we change the fuse, so it will be an issue to track the history of their sizes. Also our dashboard can't handle that many metrics well.

Reply:
My idea is if we mainly distribute the full bytecode module (I assume), then tracking the size of bytecode module is quite important, as It tells us the final binary size that will be distributed.

This size includes the VM code, workload libraries, and maybe other unexpected things. It can be unclear which part is wrong when we see the size is increased, but it also covers every part.

The next step can count the size of each part separately, which provides better insights. But it is still valuable to track the total binary size as it is the actual binary size.

Reply:
> Is there a big difference between counting `.vmfb` vs. the embedded dynamic library? The vmfb should include all the embedded dynamic libraries and I assume usually we deploy `.vmfb` (according to https://google.github.io/iree/building-from-source/riscv/#optional-configuration).

`.vmfb` includes the VM flatbuffer control and the embedded dynamic library, and the LLVM target triple should only affect the latter, but the former should affect the delta for improvement/regression. For example, person detect bytecode module built for x86_64 has the size of 729kB, but 480kB is the LLVM-target-dependent workload (model_linked_llvm_embedded_elf_x86_64.so). We want to present those two numbers separably to identify the improvement/regression.

Reply:
Indeed, all that fine-grained information could be very useful in the future but the goal for this quarter is to just track the size of the full model from the `.so` file without symbols/debug information. I expect to use this to quickly detect major changes in code size, such us the one related to `tosa.apply_scale` (one order of magnitude), and get an overall long-term view but not to scrutinize/root-cause every small increase because increases will happen. For that purpose, just tracking the full model size would be enough. We can iterate and gather more information once we have something working. Let's keep it simple for now. 

@benvanik provided some flags to keep the temporary files around [here](https://github.com/google/iree/pull/8972).